resources/definitions/os_discovery/tailyn.yaml

```diff
@@ -0,0 +1,102 @@
+modules:
+    os:
+        hardware: ENTITY-MIB::entPhysicalModelName.1
+        version: IMP-HF528-MIB::iswSdpSwVersion.0
+        features: IMP-HF528-MIB::iswSdpProductModel.0
+    processors:
+        data:
+            -
+                oid: IMP-HF528-MIB::iswCPULoading
+                num_oid: '.1.3.6.1.4.1.5833.2017.3.15.1.{{ $index }}'
+    sensors:
+        additional_oids:
+            data:
+                -
+                    oid:
+                        - IMP-HF528-MIB::iswDDMITable
+        temperature:
+            data:
+                -
+                    oid: IMP-HF528-MIB::iswDDMIA2CurrentTemperature
```

Comment:
A lot of the OIDs you are using are in the `iswDDMITable` So for example this one should be:

```
oid: IMP-HF528-MIB::iswDDMITable
value: IMP-HF528-MIB::iswDDMIA2CurrentTemperature
```

Indexes should also be the name of the oid:

```
index: 'iswDDMIA2CurrentTemperature.{{ $index }}'
```

Please update all of the sensors in line with that feedback.